docs/Contributing/API-for-contributors.md

```diff
@@ -536,54 +536,114 @@ The MDM endpoints exist to support the related command-line interface sub-comman
 - [Match preassigned profiles](#match-preassigned-profiles)
 - [Get FileVault statistics](#get-filevault-statistics)
 
-### Generate Apple DEP Key Pair
+### Generate Apple Business Manager public key (AED)
+
+`GET /api/v1/fleet/mdm/apple/dep/public_key`
 
 #### Parameters
 
-None.
+| Name | Type    | In   | Description                             |
+| ---- | ------- | ---- | --------------------------------------- |
+| alt   | string | query | **Required.** Set to "media" to download the public key for ABM. |
```

Comment:
@marko-lisica sanity check, only `'media'` is supported? any other thing is a bad request?

Reply:
Since we don't need metadata, only `media` (download) should be supported, to follow the convention we set in public API docs.